COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/CoordinateSystem.cs

```diff
@@ -6,48 +6,123 @@
 
 namespace COORDINATE_SYSTEM2
 {
+    /// <summary>
+    /// Клас Координатна система.
+    /// Містить Ліст всіх створених точок та Ліст всіх знайдених дистанцій.
+    /// Містить Add(), який створює нову точку, добавляє її в Ліст, шукає відстані 
+    /// між існуючими точками до нової точки і добавляє їх в ліст.
+    /// Містить DisplayAllPoints(), який друкує всі точки з ліста.
+    /// Містить MaxDistance() для пошуку максимальної відстані між точками.
+    /// Містить PrintAllDistances(), який друкує всі наявні відстані з Ліста.
+    /// </summary>
     internal class CoordinateSystem
     {
-        List<Point> listOfPoints = new List<Point>();
+        List<IFigure> listOfFigures = new List<IFigure>();
 
-        List<PointsDistance> pointsDistances = new List<PointsDistance>();
+        List<PointsDistance> listOfPointsDistances = new List<PointsDistance>();
 
```

Comment:
Use the plural form to make an accent that this variable is a collection. no need in `listOf` part
`listOfFigures` ->> `Figures`
`listOfPointsDistances` -> `PointsDistances`